### PR TITLE
RELEASE 1183

### PR DESCRIPTION
**Pull request by leite08**
Ref: https://github.com/metriport/metriport/issues/1183

### Description

PR: https://github.com/metriport/metriport/pull/1184

### Release Plan

- [x] publish pkg @ NPM
- [ ] merge/release this

### CODE REVIEW

File: docs/home/api-info/webhooks.mdx [hunk 1]
```diff
@@ -58,8 +57,37 @@ You can also compute an HMAC in python:
...
+message = '{"webhookUrl": "https://api.app.com/webhook"}'  # The message from the request in JSON format
+signature = 'the-request-signature'  # The signature from the header 
+
+is_signature_valid = compute_hmac(key, message, signature)
+
+if is_signature_valid:
+    print("Signature verified")
+else:
+    print("Signature verification failed")
+
```
Comment (leite08) [inline docs/home/api-info/webhooks.mdx:90]: I don't know why is this here, I think from [this merge commit](https://github.com/metriport/metriport/pull/1187/commits/1c6c5c412bc411838a44ab0f3610627ac97f6902)?

File: packages/infra/lib/api-stack.ts [hunk 2]
```diff
@@ -1114,7 +1114,7 @@ export class APIStack extends Stack {
         ...(sentryDsn ? { SENTRY_DSN: sentryDsn } : {}),
       },
       layers: [lambdaLayers.shared, lambdaLayers.chromium],
-      memory: 1024,
+      memory: 4096,
```
Comment (leite08) [inline packages/infra/lib/api-stack.ts:1117]: This too, from [this merge commit](https://github.com/metriport/metriport/pull/1187/commits/1c6c5c412bc411838a44ab0f3610627ac97f6902)

**Review by Orta21 — APPROVED** [review]

File: packages/utils/src/commonwell/cq-org-grouper.ts
Comment (RamilGaripov) [inline packages/utils/src/commonwell/cq-org-grouper.ts:1]: Seems out of the scope of this issue - not sure if planned addition

**Review by RamilGaripov — APPROVED** [review]
LGTM, with just 1 question

File: packages/utils/src/commonwell/cq-org-grouper.ts
Comment (leite08) [reply packages/utils/src/commonwell/cq-org-grouper.ts:1]: something I did for EC improvement and didn't want to lose it, also to make it available to others
